home_audit.node.js

```diff
@@ -832,27 +835,29 @@ let validationRules = {
     /*
      * hvac_heating
      */
-    heating_type_1: function(value) {
-        return this._heating_type(value, 1);
-    },
-    heating_type_2: function(value) {
-        return this._heating_type(value, 2);
-    },
-    _heating_type: function(value, num) {
-    	if(['heat_pump', 'gchp', 'mini_split'].indexOf(value) > -1 ||
-    	   ['heat_pump', 'gchp', 'mini_split'].indexOf(_homeValues['cooling_type_'+num]) > -1)
-    	{
-    		if (value !== _homeValues['cooling_type_'+num]  &&
-            _homeValues['cooling_type_'+num] !== 'none' &&
-            value !== 'none')
-        {
+    _heating_and_cooling_types: function(value, num, heatingOrCooling) {
+    	const opp = heatingOrCooling === HEATING ? COOLING : HEATING;
+    	const oppUpper = opp.charAt(0).toUpperCase() + opp.slice(1);
+    	const currUpper = heatingOrCooling.charAt(0).toUpperCase() + heatingOrCooling.slice(1);
```

Comment:
Unless I'm missing something, I think you could simplify this by just setting HEATING = 'Heating' and COOLING = 'Cooling' with uppercase first characters, since you don't use them with the lowercase first character anywhere.